index.html

```diff
@@ -1,22 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <title>Mobile Version</title>
+  <title>Francis Ugorji's Template Portfolio</title>
```

Comment:
- [ ] Remove the word template from the page title. It is much better and meaningful without it